lib/commands/install/index.ts

```diff
@@ -0,0 +1,129 @@
+
+    } catch (err) {
+      if (err instanceof CliWarn) {
+        Messenger.getInstance().warn(err.message);
+      } else if (err instanceof CliFileNotFoundError) {
+        Messenger.getInstance().warn(err.message);
+      } else if (err instanceof CliError) {
+        Messenger.getInstance().error(err.message);
+      } else {
+        Messenger.getInstance().error(err);
+      }
+      throw err;
+    }
+
+    let skillPackageSrc = ResourcesConfig.getInstance().getSkillMetaSrc(profile);
+    if (!stringUtils.isNonBlankString(skillPackageSrc)) {
+      Messenger.getInstance().error("Skill package src is not found in ask-resources.json.");
+      return;
+    }
+    if (!fs.existsSync(skillPackageSrc)) {
+      Messenger.getInstance().error(`File ${skillPackageSrc} does not exist.`);
+      return;
+    }
+
+    const componentName = cmd.componentName;
+    const cwd = process.cwd();
+
+    // Check if the component is already installed
+    const componentDirPath = path.join(
+      skillPackageSrc,
+      "skillComponents",
+      componentName
+    );
+    const configFilePath = path.join(componentDirPath, "README.md");
+
+    if (fs.existsSync(configFilePath)) {
+      Messenger.getInstance().info(
+        `Component ${componentName} is already installed. Please use the "ask update" command to update to the latest version.`
+      );
+      return;
+    }
+
+    // Install the component using npm with spinner
+    const spinner = new SpinnerView({ color: "yellow" });
+    spinner.start(`Installing ${componentName}...`);
+
+    const installCommand = spawn("npm", ["install", componentName], {
+      cwd: cwd,
+      stdio: "pipe",
+    });
+
+    installCommand.on("error", (err) => {
+      spinner.terminate(TERMINATE_STYLE.FAIL, `Failed to install ${componentName}.`);
+      throw new CliError(
+        `An error occurred while installing component ${componentName}: ${err.message}`
+      );
+    });
+
+    installCommand.on("exit", (code) => {
+      if (code !== 0) {
+        spinner.terminate(TERMINATE_STYLE.FAIL, `Failed to install ${componentName}.`);
+        throw new CliError(
+          `An error occurred while installing component ${componentName}. Please try again.`
+        );
+      } else {
+        // Copy config file to the skill package
+        const componentConfigPath = path.join(
+          cwd,
+          "node_modules",
+          componentName,
+          "README.md"
+        );
+
+        if (!fs.existsSync(componentDirPath)) {
+          fs.mkdirSync(componentDirPath, { recursive: true });
+        }
+
+        fs.copyFileSync(
+          componentConfigPath,
+          path.join(componentDirPath, "README.md")
+        );
+
+        spinner.terminate(TERMINATE_STYLE.SUCCEED, `${componentName} installed successfully! \n \u001b[1mPlease edit the README.md file copied at ${skillPackageSrc}/components/${componentName} to configure the component\n`);
```

Comment:
Can we make this more explicit that they *must* edit the `README.md` to configure the component in order to use it? This is very weakly worded as-is. 

Also it seems weird to have a configuration living in a README file, those are usually informational or instructional files, not something that's functionally important to the application...

Reply:
This was a silly mistake from me.
Since, right now we don't have a component on npm with the "config.jsonc" file, I was testing with the components already present on npm. For this, I temporarily used for the "README.md" instead of "config.jsonc"(to check if it's being copied/removed correctly). I forgot to change the file name back to "config.jsonc", will do it right now.